src/Storage/Database/Schema/Table/Users.php

```diff
@@ -21,7 +21,7 @@ protected function addColumns()
         $this->table->addColumn('lastseen',       'datetime',   ['notnull' => false, 'default' => null]);
         $this->table->addColumn('lastip',         'string',     ['length' => 32, 'default' => '']);
         $this->table->addColumn('displayname',    'string',     ['length' => 32]);
-        $this->table->addColumn('stack',          'json_array', ['length' => 1024]);
+        $this->table->addColumn('stack',          'json_array', ['length' => 1024, 'notnull' => false]);
```

Comment:
The main reason I added this, is that I was getting errors when trying to log in, that were caused when switching between 2.2 and 2.3. The value of the 'stack' column happened to be `null` on my 2.2 setup and when I switched, I was getting fatal errors, and couldn't log in. There's a screenshot in #3925